### DISCUSSION TITLE
How enrichment works?

### DISCUSSION THREAD

**Discussion by AleVirtanen** [helsenorge/structor#490] [Q&A] [answered]
I have been trying to figure out how enrichment works as you select it for textfield in structor. Do I need to add my data in contained with this kind of data? I probably don't have that how extension should work. 
{
  resourceType: 'Patient',
  name: [
    {
      use: 'official',
      family: LastName,
      given: [firstName]
    }
  ],
  // Add other fields as necessary
}

Now I don't get any value to this example item:
"item": [
    {
      "linkId": "d994a124-0423-4418-e7bc-83618a11873b",
      "type": "string",
      "text": "Test field",
      "required": false,
      "extension": [
        {
          "url": "http://ehelse.no/fhir/StructureDefinition/sdf-fhirpath",
          "valueString": "Patient.name.where(use = 'official').family"
        }
      ],
      "readOnly": true
    }
  ]

**Comment by kennethmyhra** [comment, marked as answer]
This was answered here: https://github.com/helsenorge/refero/discussions/158#discussioncomment-8382316
